### DISCUSSION TITLE
作業分担

### DISCUSSION THREAD

**Discussion by aharu6** [akiray3/r-jp-doc#27] [General]
以前から出ていたページ作業分担の話。

今現在やろうと考えていること↓
- [ ] 日本語検索対応
- [x] タブの中身htmlで表示直してみる
- [x] もらった引き数用のjsonfileで内容を直してみる
- [x] base一覧絞り込み用の　ボタン修正
- [ ]  ページトップに戻るボタン　(関数一覧のページ）
その他(なんでも)

何がやりたいですか？

**Comment by akiray3** [comment]
@aharu6 
ありがとう。頭からやってみましょうか。
まず日本語検索対応。これは昔作ったことあるから、やれそう。
and検索、or検索、引数だけに絞って検索、って感じかな。

例えば「平均」って検索したら、ちゃんと「mean」関数が出てくるようにしたいよね。
